### ISSUE TITLE
Purge Data from the UI

### ISSUE THREAD

**Issue by Chris53897** [zenstruck/messenger-monitor-bundle#22]
What do you think about a button in the UI to purge the data?
Maybe "ALL Data" is good enough.
`bin/console messenger:monitor:purge --older-than all`

https://github.com/zenstruck/messenger-monitor-bundle#messengermonitorpurge-command

**Comment by kbond** [comment]
Good idea. Thinking a _Purge_ button in the toolbar that deletes messages for the _current filter_.

**Comment by kbond** [comment]
Think we'd also need another configurable role for this operation.
